### ISSUE TITLE
Failed running test on PHPUnit 5.7

### ISSUE THREAD

**Issue by nojimage** [cakephp/cakephp#11577]
This is a (multiple allowed):

* [x] bug
* [ ] enhancement
* [ ] feature-discussion (RFC)

* CakePHP Version: 3.5.9
* PHPUnit 5.7.26

### What you did

CakePHP 3.5.9 with PHPUnit 5.7, can't running test.

### What happened

I got

```
PHP Fatal error:  Declaration of Cake\TestSuite\Fixture\FixtureInjector::startTestSuite() must be compatible with PHPUnit_Framework_TestListener::startTestSuite(PHPUnit_Framework_TestSuite $suite) in /home/travis/build/cakephp/bake/vendor/cakephp/cakephp/src/TestSuite/Fixture/FixtureInjector.php on line 29
```

It seems that a problem has occurred due to this change:
https://github.com/cakephp/cakephp/commit/a17609ab16776ace2933e34a52132f0ec436857d#diff-63e46677a3b7a874996d8017691b866dR2

PHPUnit aliases is not created because `false` is specified as the second argument of `class_exists`.

I've created a patch, how about this fix?https://github.com/nojimage/cakephp/commit/28075d82ec61fa3da242cdf2e3c9e9b6984eeed3

**Comment by nojimage** [comment]
refs https://travis-ci.org/cakephp/bake/jobs/322048601

**Comment by ADmad** [comment]
Fixed in #11578
